### PR TITLE
Initial EDA notebook

### PR DESCRIPTION
**Pull request by MichaelClifford**
This notebook contains a first pass at exploring the available sippy dataset on test failures, and provides an example for finding highly correlated tests within that data set.

### CODE REVIEW

File: notebooks/initial_EDA.ipynb [hunk 1]
```diff
@@ -0,0 +1,847 @@
+{
```
Comment (Shreyanand) [inline notebooks/initial_EDA.ipynb:1]: <p>The notebook LGTM. Just one comment:</p><p> </p><p>I found it a little hard to understand the process of finding correlation. Would it help to add a small reasoning or steps here, maybe something like</p><p>1) Encoding: Representing jobs as a vector of tests</p><p>2) Removing tests that are not frequent</p><p>3) Finding correlation matrix (which I guess would be interpreted as highly correlated tests fail together?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (MichaelClifford) [reply notebooks/initial_EDA.ipynb:1]: Thanks Shrey, I'll try to make if clearer.

**Comment by MichaelClifford** [comment]
@Shreyanand I added a bit more commentary :smiley: Please take a look and let me know if its good to merge.

**Comment by Shreyanand** [comment]
The comments make it super clear. LGTM for merging, if @durandom wants to do the honors.

**Comment by Shreyanand** [comment]
On a separate note, I don't know what the end goal here is but I just thought of this idea: What if we look at this as recommendation system problem with jobs as user and failing tests as items. So recommending failing tests to jobs. 
Maybe it'll turn out to be nonsense but preserving it here :D

**Comment by MichaelClifford** [comment]
@durandom do you want to take a look at this notebook? It provides an initial look at the ocp ci data. Otherwise, @Shreyanand feel free to merge : )

File: notebooks/initial_EDA.ipynb [hunk 2]
```diff
@@ -0,0 +1,903 @@
...
```
Comment (durandom) [inline notebooks/initial_EDA.ipynb:1]: <p>This output is tough to read. Can we render it in a table with headings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (durandom) [inline notebooks/initial_EDA.ipynb:1]: <p>Can you explain the 6.0 and why you divide it by <code>0.00980392156862745</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (durandom) [inline notebooks/initial_EDA.ipynb:1]: <p>A table render would be easier with headings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Pull request merged**